src/renderers/shared/stack/event/EventPluginRegistry.js

```diff
@@ -76,7 +87,11 @@ function recomputePluginOrdering() {
  * @return {boolean} True if the event was successfully published.
  * @private
  */
-function publishEventForPlugin(dispatchConfig, PluginModule, eventName) {
+function publishEventForPlugin(
+  dispatchConfig: Object,
```

Comment:
Can you avoid using Object. I would prefer that you create a type for it and mark it as `any` at the top of the file rather than having a partially typed variable. The rule of thumb I'm using is that it should either be a precise type or any, this way when you run flow coverage you know if you're 100% good to go or flow doesn't even try.